File: xlsx/typescript/delete/xlsx_delete.ts
// ---------------------------------------------------------------
//
//	xlsx_delete.ts
//
//					May/05/2020
//
// ---------------------------------------------------------------
var fs = require('fs')
var node_common = '/var/www/data_base/common/node_common'

var xlsx_manipulate=require (node_common + '/xlsx_manipulate')
// ---------------------------------------------------------------
console.log ("*** 開始 ***")

var xlsx_file: string = process.argv[2]
var key: string =process.argv[3]

console.log (xlsx_file)
console.log (key)

var dict_aa = xlsx_manipulate.xlsx_read_proc (xlsx_file)

if (key in dict_aa)
	{
	delete dict_aa[key]
	xlsx_manipulate.xlsx_write_proc (xlsx_file,dict_aa)
	}

console.log ("*** 終了 ***")
// ---------------------------------------------------------------

File: xlsx/python/delete/xlsx_delete.py
#! /usr/bin/python3
# -*- coding: utf-8 -*-
#
#	xlsx_delete.py
#
#					Sep/29/2016
import	sys
#
# ---------------------------------------------------------------
sys.path.append ('/var/www/data_base/common/python_common')
from text_manipulate import dict_delete_proc
from xlsx_manipulate import xlsx_read_proc
from xlsx_manipulate import xlsx_write_proc
# ---------------------------------------------------------------
#
sys.stderr.write ("*** 開始 ***\n")
#
xlsx_file = sys.argv[1]
key = sys.argv[2]
print ("%s" % key)

dict_aa = xlsx_read_proc	(xlsx_file)

dict_bb = dict_delete_proc (dict_aa,key)

xlsx_write_proc (xlsx_file,dict_bb)
#
sys.stderr.write ("*** 終了 ***\n")
# ---------------------------------------------------------------
